test/client/calendar/CalendarUtilsTest.js

```diff
@@ -163,6 +163,7 @@ o.spec("calendar utils tests", function () {
 			o(parseTime("263PM")).deepEquals(null)
 			o(parseTime("1403PM")).deepEquals(null)
 			o(parseTime("14:03:33PM")).deepEquals(null)
+			o(parseTime("9:37 acme")).deepEquals(null)
```

Comment:
good catch!

Reply:
This shouldn't have triggered an error, because there is the second check after the regex does its thing. But I didn't mean to use wildcards in the regex, so I put this in to make sure we don't regress if the second check is ever changed or removed.